packages/dds/tree/src/test/feature-libraries/sequence-field/utils.ts

```diff
@@ -861,7 +860,9 @@ function newCrossFieldTable<T = unknown>(): CrossFieldTable<T> {
 			if (addDependency) {
 				addCrossFieldQuery(getQueries(target), revision, id, count);
 			}
-			return getFromRangeMap(getMap(target).get(revision) ?? [], id, count);
+			const rangeMap = getMap(target).get(revision) ?? new RangeMap<T>();
+			// eslint-disable-next-line @typescript-eslint/no-unsafe-return
```

Comment:
Seems to compile fine without this `eslint-disable-next-line @typescript-eslint/no-unsafe-return` for me